source/background.js

```diff
@@ -24,3 +24,61 @@ browser.webRequest.onBeforeRequest.addListener(({url}) => {
 }, {
 	urls: ['https://pbs.twimg.com/media/*']
 }, ['blocking']);
+
+// Multiple Account Section
+
+function getAccount(sendResponse) {
+	chrome.cookies.get({
```

Comment:
Use the `browser` global instead whenever possible, as it returns a promise instead of using a callback, so we can use async/await. See: https://github.com/mozilla/webextension-polyfill